benchmarks/std/main.cpp

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#include <benchmark/benchmark.h>
+//
+#include <cstddef>
+
+int main(int argc, char** argv) {
+    ::benchmark::Initialize(&argc, argv);

```

Comment:
Leading `::` is unusual for this repository, used only in macros such as `_STD` or `_RANGES` but not directly in tests